app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/profile/ProfilePresenterImpl.kt

```diff
@@ -36,12 +38,14 @@ class ProfilePresenterImpl(
 
     override fun requestProfile(cookies: String, username: String) {
         view.showProgressBar(true)
-        provider.requestProfile(cookies, username, object : PresenterCallback<Any> {
-            override fun onSuccess(o: Any?) {
+        provider.requestProfile(cookies, username, object : PresenterCallback<ProfileResponse> {
+            override fun onSuccess(profileResponse: Any?) {
                 view.showProgressBar(false)
-                val response = o as? ProfileResponse
+
+                val response = profileResponse as? ProfileResponse
```

Comment:
and here

Reply:
oh! those came back when i was trying to fix conflict